Project1_tr

```diff
@@ -0,0 +1,350 @@
+1	              (1)	      ¬((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))	            A
+1	              (2)	      (¬Px ∨ ¬Qx) ∧ ¬¬(Px ∧ Qx)	            1→R
+1	              (3)     	¬Px ∨ ¬Qx	                              2∧B
+1	              (4)	      ¬¬(Px ∧ Qx)	                            2∧B
+1	              (5)	      Px ∧ Qx	                                4DN
+1	              (6)	      Px	                                     5∧B
+1	              (7)	      Qx	                                     5∧B
+8	              (8)	      ¬Px	                                      A
+8,1	            (9)       ¬Px ∧ Px                              	 8,6∧E
+10	            (10)	    ¬Qx	                                      A
+10,1	          (11)	    ¬Qx ∧ Qx	                               10,7∧E
+10,1	          (12)	    ¬Px ∧ Px	                               11 e.f.q.
+1	              (13)      ¬Px ∧ Px	                               3,8,9,10,12∨B
+                (14)	    ¬¬((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))	          1,13RAA
+                (15)	    (¬Px ∨ ¬Qx) → ¬(Px ∧ Qx)	              14DN
+
+
+(d) (¬Px ∧ ¬Qx) → ¬(Px ∨ Qx)
+
+Premisses	      Line	          Proposition	                     Rule
+1	              (1)	          ¬((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))	        A
+1	              (2)	          ¬Px ∧ ¬Qx ∧ ¬¬(Px ∨ Qx)	           1→R
+1	              (3)	          ¬Px ∧ ¬Qx	                           2∧B
+1             	(4)	          ¬¬(Px ∨ Qx)	                         2∧B
+1	              (5)	          Px ∨ Qx	                             4DN
+6	              (6)	          Px	                                  A
+1	              (7)	          ¬Px	                                  3∧B
+1	              (8)         	¬Qx	                                  3∧B
+1,6	            (9)	          ¬Px ∧ Px	                           7,6∧E
+10	            (10)	         Qx	                                  A
+1	              (11)          ¬Px	                                  3∧B
+1	              (12)	         ¬Qx	                                3∧B
+1,10	          (13)	         ¬Qx ∧ Qx	                           12,10∧E
+1,10	          (14)	         ¬Px ∧ Px	                           13 e.f.q.
+1	              (15)	         ¬Px ∧ Px	                           5,6,9,10,14∨B
+                (16)	         ¬¬((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))	      1,15RAA
+                (17)	         (¬Px ∧ ¬Qx) → ¬(Px ∨ Qx)	          16DN
+
+
+
+10. Compare and contrast the proofs provided for (a) in your answers to questions 8 and 9. Explain the different assumptions, strategies, etc. exhibited in tree proofs vs natural deduction proofs.
+
+The main difference or contrast between the tree-proofs in question 9 and the natural deduction proofs in question 10 is that the tree proofs omit the rules/justifications that signal each step of the proof. The natural deductions, by contrast, do not. In fact, the proof generator I used for question 10 annotated rather heavily.
```

Comment:
Is there anything different about the structures of the respective proofs generated?